integrationtests/src/test/java/org/axonframework/integrationtests/loopbacktest/synchronous/SynchronousLoopbackTest.java

```diff
@@ -62,7 +62,7 @@ void setUp() {
         aggregateIdentifier = UUID.randomUUID().toString();
         commandBus = SimpleCommandBus.builder().build();
         eventStore = spy(EmbeddedEventStore.builder().storageEngine(new InMemoryEventStorageEngine()).build());
-        eventStore.publish(new GenericDomainEventMessage<>("test", aggregateIdentifier, 0,
+        eventStore.publish(new GenericDomainEventMessage<>(CountingAggregate.class.getSimpleName(), aggregateIdentifier, 0,
```

Comment:
Does this test work without these changes? If so, I suggest not doing this change. If not, we probably have a backward compatibility issue.